src/expr/src/agg/general.rs

```diff
@@ -195,258 +219,401 @@ fn bool_or(state: bool, input: bool) -> bool {
 
 #[cfg(test)]
 mod tests {
```

Comment:
note: unit tests in this file are merged from `src/stream/src/executor/aggregation/agg_impl/foldable.rs`.